enzy_htp/_interface/pymol_interface.py

```diff
@@ -297,6 +307,11 @@ def export_enzy_htp_stru(self,
         sp = PDBParser()
         pymol_outfile_path = self.export_pdb(pymol_obj_name, pymol_session, if_retain_order=if_retain_order)
         res = sp.get_structure(pymol_outfile_path, allow_multichain_in_atom=True)
+
+        # update order of non-mutated residues
+        if ordered_stru is not None:
+            order_to_stru(res, ordered_stru)
```

Comment:
Sorry I kinda presented it not very clear in the issue that this functions should be called "after" export_enzy_htp_stru() is called in api.py, which means it should be in mutate_stru_with_pymol()